lib/mayaUsd/fileio/translators/translatorXformable.cpp

```diff
@@ -146,8 +148,72 @@ static void _setAnimPlugData(
     }
 }
 
+static void _applyAnimPlugData(
```

Comment:
Please add a brief description for what _applyAnimPlugData()  does.

Reply:
Certainly for another day and PR, but I had a hard time following the logics with all these `static `and `_ `stuff all over the place. All of these helper functions can go under the namespace.

```
namespace
{
    bool getXformOpAsVec3d(...)
    {
    ...
    }
    void setAnimPlugData(...)
    {
    ...
    }
    void applyAnimPlugData(...)
    {
    ...
    }
    bool isArrayVarying(...)
    {
    ...
    }

    void setMayaAttribute(...)
    {
    ...
    }

    void applyToMayaAttribute(...)
    {
    ...
    }

    bool pushUSDXformOpToMayaXform(...)
    {
    ...
    }
    
    bool isIdentityMatrix(...)
    {
    ...
    }

    bool pushUSDXformToMayaXform(...)
    {
    ...
    }
} // namespace
```

Reply:
unused variable at line 649

https://github.com/Autodesk/maya-usd/blob/f531c371d04e9c4910d36ece17c17558f3d0ff2a/lib/mayaUsd/fileio/translators/translatorXformable.cpp#L649